src/Interpreters/Context.h

```diff
@@ -511,7 +511,6 @@ class Context
     BackgroundProcessingPool & getBackgroundPool();
     BackgroundProcessingPool & getBackgroundMovePool();
     BackgroundSchedulePool & getSchedulePool();
-    BackgroundSchedulePool & getMessageBrokerSchedulePool();
```

Comment:
Interesting, it has been added in #13939 but had never been used, maybe worth use it for kafka instead of removing?